.github/file-filters.yml

```diff
@@ -1,3 +1,17 @@
-backend:
-  - '**/*.py'
-  - 'requirements*.txt'
+# This is used by the action https://github.com/dorny/paths-filter (which we have forked to https://github.com/getsentry/paths-filter)
+
+# TODO: There are some meta files that we potentially could ignore for both front/backend,
+# as well as some configuration files that should trigger both
+
+# backend == !frontend
+
+frontend:
```

Comment:
I tested these locally, at the top you can see the list of files that were modified and then the list of filtered files

![image](https://user-images.githubusercontent.com/79684/96799178-620e9d80-13b7-11eb-81d3-99d5574045bb.png)

Reply:
Oh, cool.